src/widgets/medications/medication-order-basket.component.tsx

```diff
@@ -167,8 +170,30 @@ export default function MedicationOrderBasket(
     );
   };
 
+  const closeForm = () => {
+    let userConfirmed: boolean = false;
+    if (formChanged) {
+      userConfirmed = confirm(
+        "There is ongoing work, are you sure you want to close this tab?"
+      );
+    }
+
+    if (userConfirmed && formChanged) {
+      props.entryCancelled();
+      props.closeComponent();
+    } else if (!formChanged) {
+      props.entryCancelled();
+      props.closeComponent();
+    }
+  };
+
   return (
-    <div className={styles.medicationOrderBasketContainer}>
+    <div
+      className={styles.medicationOrderBasketContainer}
+      ref={medsRef}
+      onChange={evt => setFormChanged(true)}
```

Comment:
Does this handler really fire? Its placed on a `div`.

Reply:
removed this since its not working